packages/babel-plugin-transform-classes/test/fixtures/loose/accessing-super-class/output.js

```diff
@@ -13,14 +13,14 @@ function (_Foo) {
     woops.super.test();
     _this = _Foo.call(this) || this;
 
-    _Foo.prototype.test.call(_this);
```

Comment:
I would think loose mode doesn't need this assertion right?

Reply:
It's closer to spec code, but this is only here because I haven't optimized any special cases yet.